packages/zowe-explorer/package.json

```diff
@@ -699,6 +708,16 @@
           "command": "zowe.uss.removeSavedSearch",
           "group": "inline"
         },
+        {
+          "when": "view == zowe.uss.explorer && viewItem =~ /^(?!(directory|ussSession|favorite|profile_fav))/",
+          "command": "zowe.uss.refreshUSS",
+          "group": "0_mainframeInteraction"
+        },
```

Comment:
@rudyflores Please remove this block (lines 711-715), as `zowe.uss.refreshUSS` is already declared with the same `when` clause further down (lines 736-740). It is causing the refresh ("Pull from mainframe") command to appear twice for files:

<img width="248" alt="Screen Shot 2021-06-23 at 10 47 35 AM" src="https://user-images.githubusercontent.com/45975633/123118334-8a500f80-d410-11eb-8ab7-839fee48702b.png">